app/my-bets/page.tsx

```diff
@@ -7,96 +7,131 @@ import { Market, Position, tradesOutcomeBalance } from '@/entities';
 
 import { getUserPositions } from '@/queries/conditional-tokens';
 import { UserPosition } from '@/queries/conditional-tokens/types';
-import { getConditionMarket, getMarketUserTrades } from '@/queries/omen';
-import { useQuery, useQueryClient } from '@tanstack/react-query';
-
-import { PropsWithChildren, ReactNode, useCallback, useEffect, useState } from 'react';
+import {
+  Condition,
+  FixedProductMarketMaker,
+  FpmmTrade,
+  getConditionMarket,
+  getMarketUserTrades,
+} from '@/queries/omen';
+import { useQuery } from '@tanstack/react-query';
+
+import { PropsWithChildren, ReactNode, useMemo } from 'react';
 import { TabBody, TabGroup, TabHeader, TabPanel, TabStyled } from '@swapr/ui';
 import { useAccount } from 'wagmi';
 
+export interface UserPositionComplete extends UserPosition {
+  fpmmTrades: FpmmTrade[];
+  market: FixedProductMarketMaker;
+  condition: Condition;
+}
+
+const sortByNewestBet = (a: any, b: any) => {
```

Comment:
We can type it, right?

```suggestion
const sortByNewestBet = (a: UserPositionComplete, b: UserPositionComplete) => {
```